apps/web/src/app/(admin)/backoffice/_components/BackofficeTabs/index.tsx

```diff
@@ -2,7 +2,7 @@
 
 import { ReactNode } from 'react'
 
-import { TabSelector } from '@latitude-data/web-ui'
+import { TabSelector } from '@latitude-data/web-ui/molecules/TabSelector'
```

Comment:
instead of exporting a index.ts file we export each component individually which allows nextjs to tree shake the chunks and results also in faster dev times

Reply:
Nice

Reply:
So in the end we keep not compiling packages separately no?

Reply:
yup, no compilation required since we don't have workspace packages anymore